scripts/generate_daily_article.py

```
@@ -149,8 +149,28 @@ async def main_async(args: argparse.Namespace):
             trends = SAMPLE_TRENDS
             automation = None
         else:
-            automation = ContentAutomationService(db, groq_api_key)
-            trends = await automation.analyze_trends(source_data)
+            try:
+                automation = ContentAutomationService(db, groq_api_key)
+                trends = await automation.analyze_trends(source_data)
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)